dm/pkg/schema/tracker.go

```diff
@@ -141,7 +167,12 @@ func (tr *Tracker) Init(
 		tableInfos:     make(map[string]*DownstreamTableInfo),
 	}
 	// TODO: need to use upstream timezone to correctly check literal is in [1970, 2038]
-	se := executorContext{Context: mock.NewContext()}
+	seContext := mock.NewContext()
+	seContext.Store = &trackerMockStore{}
```

Comment:
/cc @okJiang 

can you check this change is expected? Can we move it into mock.NewContext()

Reply:
i think this should be moved into `mock.NewContext()` or make `GetBDRMode()` not requiring a valid `txn` but that would involve updating TiDB first 🥲

Reply:
I think it is unnecessary?
<img width="1361" alt="image" src="https://github.com/pingcap/tiflow/assets/23556834/c85cb783-94fd-457e-a897-cc5bf2364fd3">

https://github.com/pingcap/tidb/pull/50939/files

Reply:
But for v7.6, the pr is not merged.

Reply:
> can you check this change is expected? Can we move it into mock.NewContext()

Yes

Reply:
I just see it targets to release-7.6. We don't need to maintain non LTS versions 😂  If there's an user request I prefer we cherry-pick https://github.com/pingcap/tidb/pull/50939 to TiDB's release-7.6 branch

Reply:
if pingcap/tidb#50939  can be cherry-picked to 7.6 I'd prefer that ;)